elasticsearch/linux_distributions/opendistro-tar-build.sh

```diff
@@ -125,7 +137,7 @@ rm -rf $PACKAGE_NAME-$OD_VERSION
 ls -ltr $TARGET_DIR
 tar_artifact=`ls $TARGET_DIR/*.tar.gz`
 tar_checksum_artifact=`ls $TARGET_DIR/*.tar.gz.sha512`
-aws s3 cp $tar_artifact s3://$S3_BUCKET/downloads/tarball/opendistro-elasticsearch/
-aws s3 cp $tar_checksum_artifact s3://$S3_BUCKET/downloads/tarball/opendistro-elasticsearch/
-aws cloudfront create-invalidation --distribution-id E1VG5HMIWI4SA2 --paths "/downloads/*"
+echo "Staging destination : s3://$S3_RELEASE_BUCKET/${PLUGIN_PATH}${OD_VERSION}/odfe/"
+aws s3 cp $tar_artifact s3://$S3_RELEASE_BUCKET/${PLUGIN_PATH}${OD_VERSION}/odfe/
+aws s3 cp $tar_checksum_artifact s3://$S3_RELEASE_BUCKET/${PLUGIN_PATH}${OD_VERSION}/odfe/
```

Comment:
Can you move this out of the script? Add it in the workflow? For all other workflows too it is outside any script which makes it visible for any testers

Reply:
Not moving the cp command to workflow as per our discussion. We can later modify the script to include a argument for a dry run.